pyproject.toml

```diff
@@ -1,5 +1,9 @@
-[tool.pytest.ini_options]
+[build-system]
+requires = ["setuptools", "setuptools-rust"]
```

Comment:
This is what the website for setuptools rust suggested: https://pypi.org/project/setuptools-rust/